pkg/controllers/provisioning/provisioner.go

```diff
@@ -74,7 +74,7 @@ func (p *Provisioner) provision(ctx context.Context) (err error) {
 	defer func() {
 		for i := 0; i < len(pods); i++ {
 			select {
-			case p.results <- err: // Block until result is communicated
+			case p.results <- struct{}{}: // Block until result is communicated
```

Comment:
`results` seems like a slightly confusing name (especially since there's also a `done` channel right next to it), since no "results" are actually being communicated (e.g., about success/failure/status)?

Reply:
Agree. I need to completely re-write most of this to get the scheduling logic out of this controller. Think of the current impl as an interim step to get multiple provisioners working. Can we address this during the refactor? You can see a preview here: https://github.com/aws/karpenter/pull/842